pegasus/sites.v3/code.org/public/ai/index.haml

```diff
@@ -127,12 +127,13 @@ theme: responsive_full_width
   .wrapper.centered
     %h2.heading-sm
       =hoc_s("ai_donors.desc")
-    .logos-wrapper.no-margin-bottom
+    .logos-wrapper.ai-logos-wrapper.no-margin-bottom
```

Comment:
I checked this out locally, and you'll can just add these inline styles on the wrapper and remove the styles you added in `ai-styles.scss`.  I usually avoid this, but in this one-off case it's fine and saves lines 😅 
```suggestion
    .logos-wrapper.no-margin-bottom{style: "max-width: 950px; margin-inline: auto;}
```

<img width="1061" alt="Screenshot 2024-09-19 at 8 41 36 AM" src="https://github.com/user-attachments/assets/a8d548ac-b034-47df-8cd7-b89453da6841">

----
Removing the `flex-wrap` breaks this on mobile.

<img width="300" alt="Screenshot 2024-09-19 at 8 39 12 AM" src="https://github.com/user-attachments/assets/b4d74177-4226-4582-86ce-8b113720e2d2">

Reply:
Why use inline styles over class styles? I thought the latter was generally preferred?

Reply:
And where did 950px come from?

Reply:
Since this is a one-off exception for an otherwise widely used element it's fine to add it inline. If you prefer to add it to the scss file you can replace the existing styles under the `.logos-wrapper` selector with these two new ones.

And 950px breaks the container at a point where two logos will go to the second line, otherwise only one will drop and that looks less balanced.

Reply:
Sorry for all the questions -- just want to understand best practices here :)

I'll probably add inline just to clear this PR, which I've been dragging on, but still curious!

Reply:
You could replace them in the `ai-styles.scss` that's connected to this page, but yeah inline is fine for now!

Reply:
oo gotcha gotcha! Thanks!